Samples/ProjectTracker/ProjectTracker.AppServerCore/Startup.cs

```diff
@@ -6,6 +6,9 @@
 using Microsoft.Extensions.Hosting;
 using Microsoft.AspNetCore.Server.Kestrel.Core;
 using ProjectTracker.Configuration;
+//using Microsoft.AspNetCore.ResponseCompression;
```

Comment:
I think I'd rather just turn on compression in ProjectTracker rather than leaving these as comments. Might as well directly show folks the "right way" to set this up?

Reply:
If you're about to do that, then installing Microsoft.AspNetCore.ResponseCompression is a must. I didn't include it in PR.

Reply:
That needs to be a dependency added to the app server? Also to the various client projects?

Reply:
No, just add NuGet Microsoft.AspNetCore.ResponseCompression to AppServerCore.

I will do another PR today.

Reply:
I see where you uncommented the code, but not where the nuget package was added to the appservercore csproj file.

Reply:
Well, I guess you are right, the package was not needed. I remember adding it to my project (not the project tracker).

This seems fine as is.